packages/whats-new/src/index.js

```diff
@@ -31,17 +30,15 @@ const WhatsNewGuide = () => {
 			} );
 	}, [ locale ] );
 
-	const toggleWhatsNew = () => setShowGuide( false );
-
-	if ( ! ( whatsNewData && showGuide ) ) return null;
+	if ( ! whatsNewData ) return null;
```

Comment:
Nit: according to the WP coding standards, we always wrap code blocks in curly braces, even if they contain one line of code only. 
```suggestion
	if ( ! whatsNewData ) {
		return null;
	}
```